MainApp/MainApp/Feature/Profile/ProfileLandingController.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  ProfileLandingController.swift
+//  MainApp
+//
+//  Created by Owen Prasetya on 29/07/20.
+//  Copyright © 2020 William Inx. All rights reserved.
+//
+
+import UIKit
+import Component
+
+class ProfileLandingController: MasterViewController, ProfileLandingViewProtocol, UITableViewDelegate, UITableViewDataSource {
```

Comment:
refactor tableviewDelegate and datasource to its own file, and get the data from viewModel